lib/Minz/Request.php

```diff
@@ -294,8 +348,41 @@ private static function magicQuotesOff() {
 		}
 	}
 
+	/**
+	 * @return array
+	 */
+	private static function getHeaders() {
```

Comment:
I would rename this method: it's too close of `getHeader` which does something completely different. What do you think of `initHeaders`? Or `setHeaders`, taking an array as an argument (in this case, `$_SERVER`)?